src/oss.yml

```diff
@@ -24,6 +24,8 @@ paths:
     $ref: "./common/paths/authorizations.yml"
   /authorizations/{authID}:
     $ref: "./common/paths/authorizations_authID.yml"
+  /authorizations/{authID}/password:
```

Comment:
This needs a `servers` to set the `/private/legacy` prefix, yes? Otherwise it'll fall under the default `/api/v2` prefix.